clar2wasm/src/standard/standard.wat

```diff
@@ -1468,6 +1465,40 @@
         (i32.store offset=28 (local.get $origin) (i32.add (i32.load offset=28 (local.get $origin)) (local.get $h)))
     )
 
+    ;;
+    ;; logical not implementation
+    ;;
+    (type $not-func (func (param $bool_in i32) (result i32)))
```

Comment:
For consistency, let's move this up to the top of the file, with the other types.